domains/shivang.json

```diff
@@ -0,0 +1,12 @@
+{
+  "description": "Projects by Shivang Kakkar",
+  "repo": "https://github.com/ShivangKakkar/shivangkakkar.github.io",
+  "owner": {
+    "username": "ShivangKakkar",
+    "email": "shivangkakkar88@gmail.com",
+    "telegram": "ShivangKakkar88"
+  },
+  "record": {
+    "CNAME": "ShivangKakkar.github.io"
```

Comment:
404?

Reply:
I'm sorry about that. I forgot to remove the old one. It's visible now.